geoapps/drivers/components/locations.py

```diff
@@ -14,22 +14,41 @@
 
 
 class InversionLocations:
-    """ Retrieve topography data from workspace and apply transformations. """
+    """
+    Retrieve topography data from workspace and apply transformations.
 
-    def __init__(self, workspace, params, mesh, window):
+    Methods
+    -------
+    get_locs: Returns locations of data object centroids or vertices.
+    filter: Apply accumulated self.mask to array, or dict of arrays.
+    rotate: Un-rotate data using origin and angle assigned to inversion mesh.
+
+    """
+
+    def __init__(self, workspace, params, window):
```

Comment:
I can't import InversionWindow to type the window parameter because it causes a circular import.  This fix seems to work, but adds a bunch of junk to the import section, thoughts?: 

https://stackoverflow.com/questions/39740632/python-type-hinting-without-cyclic-imports

Reply:
Overhauled the typing to use this method.